src/components/RecentPosts/Post/Post.style.js

```diff
@@ -0,0 +1,47 @@
+"use client"
+
+import Image from "next/image"
+import Link from "next/link"
+import styled from "styled-components"
+
+export const Date = styled.p`
+  font-size: ${({ theme }) => theme.fontSizes.xsmall};
+  color: ${({ theme }) => theme.colors.secondary};
+`
+
+export const Description = styled.p`
+  font-size: ${({ theme }) => theme.fontSizes.small};
+  color: ${({ theme }) => theme.colors.secondary};
+  text-align: left;
+`
+
+export const StyledImage = styled(Image)`
```

Comment:
I don't think you need a styled-component for this. You can set width and height directly